config/locales/en.yml

```diff
@@ -353,6 +353,8 @@ en:
     unprocessable_entity: We're unable to process your request.
     not_found: Page not found.
     server_error: Something went wrong at our end.
+    external_server_error:
+      dfe_sign_in: There was a problem with DfE Sign-in, please try again in a moment.
```

Comment:
I made this up as Michael wasn't available. If we feel it's not good enough to start with, let's flag it.